_drafts/release-osiam-3.0.md

```diff
@@ -3,81 +3,124 @@ title:  "Release: OSIAM 3.0"
+But we also removed custom or outdated APIs, like the Facebook Connect emulation and the PATCH API.
+In terms of security, the most important change is the automatic migration to bcrypt hashes for passwords.
 
 You can download OSIAM using the following link:
-[osiam-3.0.war](https://github.com/osiam/osiam/releases/download/3.0/osiam-3.0.war)
-
-It follows an overview of the most important changes.
-See the full changelog at the [OSIAM 3.0 release page on GitHub](https://github.com/osiam/osiam/releases/tag/3.0).
-
-## SCIM 2.0 Compliance
-
-TODO: Write me
-
-### Removal of PATCH
-
-TODO: Write me
-
-### `/Me` Endpoint
-
-TODO: Write me
-
-### Misc
-
-TODO: Write me
-
-- Reply with 400 BAD REQUEST to invalid filters.
-- Change URL of service provider configuration resource from
-  `/ServiceProviderConfigs` to `/ServiceProviderConfig`.
-- Always return the `id` attribute, when searching for `User`s.
-- Allow colons (:) as field separators for URNs of extensions, since this is
-  what the SCIM specification defines. Using periods (.) is still possible,
-  but will log a warning message.
-- Fields of the core schemas for user and group can be fully qualified, i.e.
-  `filter=urn:ietf:params:scim:schemas:core:2.0:User:userName sw "J"`.
-
-## Security
-
-TODO: Write me
 
-### BCrypt Password Hashing
+[osiam-3.0.war](https://dl.bintray.com/osiam/downloads/osiam/3.0/osiam-3.0.war)
+([GPG Signature](https://dl.bintray.com/osiam/downloads/osiam/3.0/osiam-3.0.war.asc))
 
-TODO: Write me
-
-### Disable Password Searches
-
-TODO: Write me
-
-### Removal of Method-Based Scopes
-
-TODO: Write me
+It follows an overview of the most important changes.
+See the full changelog of [OSIAM 3.0](https://github.com/osiam/osiam/blob/3.0/CHANGELOG.md) for more information.
 
 ## Deployment and Configuration
 
-TODO: Write me
-
-### YAML configuration
-
-TODO: Write me
-
-### Merge of the Servers
+**NOTE:** This part of OSIAM changed heavily, making it incompatible with any previous version.
+For a complete overview of the necessary steps to update, see the [migration notes](https://github.com/osiam/osiam/blob/master/docs/migration.md#from-25-to-30).
+It's also recommended to read the complete [installation guide](https://github.com/osiam/osiam/blob/master/docs/detailed-reference-installation.md)
+once again to fully understand the new features and changes.
+
+Until now, OSIAM has been distributed as a bunch of distinct services.
+There were the auth-server and resource-server, that formed the required core of OSIAM.
+And there were the add-ons, that extended OSIAM with an administration, a registration process and much more.
+With OSIAM 3.0, this is not the case anymore.
+We've decided to make OSIAM a single application again.
+You can get more information about our decision in the corresponding [GitHub issue](https://github.com/osiam/osiam/issues/43).
+As this is a complex undertaking, we started it with just merging the core services,
```

Comment:
as this is a complex undertaking, the start was made with only the core services